third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -285,6 +319,27 @@ resource "google_composer_environment" "test" {
 `, name)
 }
 
+func testAccComposerEnvironment_private(name string) string {
```

Comment:
Was there an issue with testing the other fields (like range overlap)? Would be a good idea to test actually setting the other fields for cidr ranges if we can